employees/templates/employees/author_report_list.html

```diff
@@ -17,11 +17,9 @@
 <h1>{{ object.email }} {{ title_date }}{{ UI_text.PAGE_TITLE.value }}</h1>
 <a href="{% url 'custom-users-list' %}" class="btn btn-primary hidden-print">
     {{ UI_text.RETURN_BUTTON_MESSAGE.value }}
-    {% with date=path|extract_year_and_month_from_url %}
-        <a href="{% url 'export-data-xlsx' pk=object.id year=date.0 month=date.1 %}" class="btn btn-info">
+        <a href="{% url 'export-data-xlsx' pk=object.id year=year month=month %}">
```

Comment:
Unfortunately this is not working. This will return reports from current month. This is serious bug and we can't merge it until this will be fixed.

Reply:
As I said on rocket I find another solution

Reply:
Good catch, any idea why this happened? Probably related somehow to this change https://github.com/Code-Poets/sheetstorm/pull/356/files#diff-f15a2f3cdb6b490acf1f4ccc5a64624fR119 but it looks pretty straightforward, are the values passed correctly from url?